src/api.ts

```diff
@@ -126,7 +126,7 @@ export function create(req: Request): Promise<Response> {
 					})
 					.tap(service => {
 						// Create images and attach labels and env vars
-						return createImage(api, res.release.id, serviceDescription.labels, serviceDescription.environment, {
+						return createImage(api, res.release.id, serviceDescription.labels!, serviceDescription.environment!, {
```

Comment:
Would it be better to do `|| {}` on these?

Reply:
I think I will actually just update the typings of the `createImage()` to also allow undefined, since `_.toPairs()` does work with that as well. Thanks for mentioning this.